package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "ios-detector",
+  "version": "0.0.0",
```

Comment:
Symver suggests starting at `0.1.0`. See first question in the [FAQ](http://semver.org/).